app/src/renderer/vuex/modules/wallet.js

```diff
@@ -51,6 +51,7 @@ export default ({ node }) => {
       commit(`setWalletAddress`, address)
       await dispatch(`queryWalletBalances`)
       dispatch(`loadDenoms`)
+      dispatch(`getTotalRewards`)
```

Comment:
you can instead use a `initializeWallet` action in the `distribution` module which will in parallel be triggered. this way the code is more modular and domain specific.

Reply:
`initializeWallet` it's not distribution module specific

Reply:
I'd prefer to call everything on the original `initializeWallet` action

Reply:
let's rename `initializeWallet` to `initializeAccount` or `initializeSession` then